release_notes.md

```diff
@@ -37,3 +37,16 @@
 
 - Added support for OpenAI assistant
 
+- The AI Assistant can now understand and respond to context sensitive conversations and perform given SOC tasks.
+
+    - With bot's enhanced understanding of the natural language, it now derives its operating modes from the conversation; making the options &mdash; Playbook Blocks and Conversation Mode &mdash; redundant.
+
+    - The bot now has increased access to various modules, maintaining privacy is paramount. Sensitive data is still being actively masked for all interactions with the LLM.
+
+    - The bot can now hold conversations to understand requirements and generate a more targeted playbook.
+
+    - Through enhanced comprehension, the bot can also navigate to modules, its records, and playbook logs; in addition to alert investigation, remediation, escalation to incident, and even setting up war rooms, among several others.
```

Comment:
in addition to alert investigation, remediation, --> these are some soc use-cases are introduced in this release hence needs more attention.

Reply:
Done